keepassxc-browser/content/fields.js

```diff
@@ -416,13 +416,19 @@ kpxcFields.isTopElement = function(elem, rect) {
         return false;
     }
 
-    // Check topmost element from three points inside the input
-    const verticalMiddle = rect.top + (rect.height / 2);
     const rootNode = elem.getRootNode() ?? document;
+
+    // Returns the topmost element from point x,y. If the input has a label as the top element, allow it.
+    const getTopmostElement = (x) => {
+        const topElement = rootNode.elementFromPoint(x, rect.top + (rect.height / 2));
```

Comment:
Rect isnt captured by this inner function, is it implied that the parent scope rect is accessible here?

Reply:
Yes. I can add the variable to the function for better readability. I should probably also check (just to be sure) that the label element is smaller than the input size.